src/frontend/magnify/packages/components/src/components/jitsi/MagnifyMeeting/index.tsx

```diff
@@ -29,15 +30,24 @@ export const MagnifyMeeting = ({
 
   // Users
   const [users, setUsers] = React.useState<any[]>([]);
+  const setSubject = (): void => {
```

Comment:
Can you just add a comment to explain briefly what is `subject` and `localSubject` in Jitsi ?

Reply:
Deleted because we can put it in the config